src/main.cpp

```diff
@@ -2857,6 +2864,7 @@ bool ProcessNewBlock(CValidationState &state, CNode* pfrom, CBlock* pblock, bool
 bool TestBlockValidity(CValidationState &state, const CBlock& block, CBlockIndex * const pindexPrev, bool fCheckPOW, bool fCheckMerkleRoot)
 {
     const CChainParams& chainparams = Params();
+    const Consensus::Params& consensusParams = chainparams.GetConsensus();
```

Comment:
Having the chainparams variable already (which should eventually beome a parameter of the function), I would just directly use chainparams.GetConsensus() instead of defining a new variable.